templates/web/base/admin/user-alerts.html

```diff
@@ -0,0 +1,52 @@
+[% IF user.alerts %]
+<h3>[% loc("User's alerts") %]</h3>
+<form method="POST">
+<input type="hidden" name="token" value="[% csrf_token %]" >
+<input type="hidden" name="update_alerts" value="1" >
+<table>
+<tr>
+    <th>ID</th>
+    <th>Type</th>
+    <th>Parameters</th>
+    <th>Confirmed</th>
+    <th>State</th>
+    <th>Details</th>
+    <th>Enable</th>
+    <th>Disable</th>
+    <th>Delete</th>
+</tr>
+[% FOREACH alert IN user.alerts %]
+<tr>
+    <td>[% alert.id %]</td>
+    <td>[% alert.alert_type.ref %]</td>
+    <td>[% alert.parameter %] [% alert.parameter2 %]</td>
+    <td>[% IF alert.confirmed %][% loc('Yes') %][% ELSE %][% loc('No') %][% END %]</td>
+    <td>Subscribed: [% alert.whensubscribed %]
+        <br>Disabled: [% alert.whendisabled %]
+    </td>
+    <td>
+    [% SWITCH alert.alert_type.ref %]
+        [% CASE 'new_updates' %]
+            New updates on report <a href="[% c.uri_for( '/report', alert.parameter ) %]">[% alert.parameter %]</a>
+        [% CASE 'local_problems' %]
+            New problems near <a href="[% c.uri_for( '/around', { lon => alert.parameter, lat => alert.parameter2 } ) %]">[% alert.parameter %],[% alert.parameter2 %]</a>
+        [% CASE 'ward_problems' %]
+            [% body = alert.parameter %]
+            [% ward = alert.parameter2 %]
+            New problems for [% IF alert_areas.$ward %]<a href="[% c.uri_for('/reports', body_names.$body, alert_areas.$ward.name ) %]">[% alert_areas.$ward.name %]</a> ward[% ELSE %]ward id [% alert.parameter2 %][% END %] in [% IF body_names.$body %]<a href="[% c.uri_for('/reports', body_names.$body) %]">[% body_names.$body %]</a>[% ELSE %] body id [% body %][% END %]
+        [% CASE 'council_problems' %]
+            [% body = alert.parameter %]
+            New problems for [% IF body_names.$body %]<a href="[% c.uri_for('/reports', body_names.$body) %]">[% body_names.$body %]</a>[% ELSE %][% body %][% END %]
+        [% CASE 'area_problems' %]
+            New problems for area id [% alert.parameter %]
+    [% END %]
+    </td>
+    <td><input type="checkbox" name="enable_alert[[% alert.id %]]" value="[% alert.id %]"[% ' disabled' UNLESS alert.whendisabled %]></td>
```

Comment:
I'm not sure checkboxes are the right UI here, given you won't want to pick more than one - radio buttons, perhaps?